packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -40,7 +40,11 @@ export class Checkboxes extends GOVUKFrontendComponent {
     /** @satisfies {NodeListOf<HTMLInputElement>} */
     const $inputs = $module.querySelectorAll('input[type="checkbox"]')
     if (!$inputs.length) {
-      return this
+      throw new ElementError(null, {
+        componentName: 'Checkboxes',
+        identifier: '<input type="checkbox">',
+        expectedType: HTMLElement
```

Comment:
Love seeing this!

I suppose it would be `HTMLInputElement` but `null` **element** skips `expectedType` 🤷‍♂️

```suggestion
        identifier: '<input type="checkbox">'
```